style.css

```diff
@@ -11,12 +11,64 @@ body {
   margin: 0%;
 }
 
-.heading-desktop {
-  display: none;
+nav {
+  display: flex;
+  flex-direction: row;
```

Comment:
- [OPTIONAL] Your menu navigation works great, :+1: :+1: I'll suggest you add a smooth scroll to your portfolio, it will make scrolling to different sections of the page when a user clicks on a menu item smooth. You can achieve this by adding this line of CSS at the top of your CSS file. `html {
  scroll-behavior: smooth;
}`. :+1: :smile: